scientific_library/tvb/simulator/models/epileptor.py

```diff
@@ -288,8 +288,7 @@ class Epileptor(ModelNumbaDfun):
     cvar = numpy.array([0, 3], dtype=numpy.int32)  # should these not be constant Attr's?
     cvar.setflags(write=False)  # todo review this
 
-    def _numpy_dfun(self, state_variables, coupling, local_coupling=0.0,
-             array=numpy.array, where=numpy.where, concat=numpy.concatenate):
+    def dfun(self, x, c, local_coupling=0.0):
```

Comment:
In this model (and few others bellow) I see that you propose to remove the `_numpy_dfun ` method.
Indeed, that method is not directly used, but I suggest to keep it, as it is a nice fallback in case the numba dependency does not  exist. 
As for the equations that you are after for this bug, I suggest you move the documentation content from under `_numpy_dfun` on the `dfun` to be consistent to the way we read it, and all the other models.